docs/book/src/introduction.md

```diff
@@ -25,6 +25,14 @@ This book documents ClusterAPI v1.4. For other Cluster API versions please see t
 * [release-0-4.cluster-api.sigs.k8s.io](https://release-0-4.cluster-api.sigs.k8s.io)
 * [release-0-3.cluster-api.sigs.k8s.io](https://release-0-3.cluster-api.sigs.k8s.io)
 
+> ⚠️ Break Changes ⚠️
+>
+> API Version v1alpha3 support will be removed with the upcoming release of v1.5
```

Comment:
Question: is it "with the" or "in the"?

Reply:
I think either can work here. I'll change to `in the`